models/gmx/arbitrum/glp/gmx_arbitrum_glp_components.sql

```diff
@@ -0,0 +1,926 @@
+{{ config(
+        alias = 'components',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        unique_key = ['minute'],
```

Comment:
Are you 100% sure there are never going to be any duplicates if we only use `'minute'` as the unique key ? Would it make sense to add the `'amount'` in there as well ?

Reply:
The query uses: [`explode(sequence(TIMESTAMP '2021-08-31 08:13', CURRENT_TIMESTAMP, INTERVAL 1 minute))`](https://github.com/duneanalytics/spellbook/pull/1665/files#diff-35ece43ccefc27bae1364e223e1bb0fccf6c3920f62c3b47e5c8e17319078bb9R16) to generate the primary key for this table, and all subsequent data manipulations are `LEFT JOINED` into the the minute column, so there should not be any duplicates.

Apart from the minute column, there is no column with unique data in this abstraction.

Furthermore I added a unit test [here](https://github.com/duneanalytics/spellbook/pull/1665/files#diff-9dad9abe64cc3cebb8cdefdfd4094ef06f9042b2fdd56a6e1560960768889b01R18) that test the minute column for uniqueness, does that help?

So I cant add a second unique key as such, but. I can concatenate 2 random column like this : `unique_key="minute||'-'||weth_available_assets"` to generate a stronger unique_key?

Let me know if you want me to do same.

Reply:
@1chioku No it's ok, I tested it a couple of times and it looks good to me
Just add your info in the dbt_project.yml file please and we're good 👍